src/api/spec/controllers/webui/request_controller_spec.rb

```diff
@@ -38,4 +38,37 @@
       expect(response).to redirect_to(user_show_path(User.current))
     end
   end
+
+  describe "POST #delete_request" do
+    before do
+      login(submitter)
+    end
+
+    context "a valid request" do
```

Comment:
We usually prefer having one expectation in each `it`, you can use a `before` block with the post and the @bs_request.